pinot-spi/src/main/java/org/apache/pinot/spi/trace/Tracing.java

```diff
@@ -254,14 +260,25 @@ private ThreadAccountantOps() {
     }
 
     public static void setupRunner(String queryId) {
+      setupRunner(queryId, ThreadExecutionContext.TaskType.SSE);
+    }
+
+    public static void setupRunner(String queryId, ThreadExecutionContext.TaskType taskType) {
       Tracing.getThreadAccountant().setThreadResourceUsageProvider(new ThreadResourceUsageProvider());
-      Tracing.getThreadAccountant().createExecutionContext(queryId, CommonConstants.Accounting.ANCHOR_TASK_ID, null);
+      Tracing.getThreadAccountant()
+          .createExecutionContext(queryId, CommonConstants.Accounting.ANCHOR_TASK_ID, taskType, null);
     }
 
     public static void setupWorker(int taskId, ThreadResourceUsageProvider threadResourceUsageProvider,
         ThreadExecutionContext threadExecutionContext) {
+      setupWorker(taskId, ThreadExecutionContext.TaskType.SSE, threadResourceUsageProvider, threadExecutionContext);
+    }
+
+    public static void setupWorker(int taskId, ThreadExecutionContext.TaskType taskType,
```

Comment:
Can we add some javadoc saying explaining these methods? Specially saying that the one without TaskType defaults to SSE.

Reply:
Done